discovery/seata-discovery-nacos/src/main/java/io/seata/discovery/registry/nacos/NacosRegistryServiceImpl.java

```diff
@@ -44,12 +43,14 @@
 public class NacosRegistryServiceImpl implements RegistryService<EventListener> {
     private static final String DEFAULT_NAMESPACE = "";
     private static final String DEFAULT_CLUSTER = "default";
+    private static final String DEFAULT_GROUP = "SEATA_GROUP";
```

Comment:
DEFAULT_GROUP

Reply:
high and low version compatibility needs to be considered

Reply:
set default value is DEFAULT_GROUP?

Reply:
> set default value is DEFAULT_GROUP?

yes,backward compatible